src/commands/serve.ts

```diff
@@ -124,3 +132,28 @@ export const handler = async (argv: Arguments<Options>): Promise<void> => {
   await ad4mCore.initLanguages();
   console.log("All languages initialized.");
 };
+
+async function fetchLatestBootstrapSeed(appDataPath: string) {
+  return new Promise(async (resolve, reject) => {
+    const response = await fetch("https://api.github.com/repos/perspect3vism/ad4m-seeds/releases/latest")
```

Comment:
Should we use a fixed prod version instead? the compatibility between versions is a big issue holochain and ad4m haven't solved yet.